cobaya/likelihoods/base_classes/planck_clik.py

```diff
@@ -360,7 +345,7 @@ def install_clik(path, no_progress_bars=False):
 
 def get_product_id_and_clik_file(name):
     """Gets the PLA product info from the defaults file."""
-    defaults = get_default_info(name, kinds.likelihood)
+    defaults = get_default_info(name, "likelihood")
```

Comment:
Not grabbed from conventions now?

Reply:
This is a bit annoying.. we could certainly still use a kinds namedtuple (btw, I think StrEnum is coming in py10, which might be what we really want) - however none of the usual static checkers seem to then be happy using kinds.likelihood in a typed dict (needs the finalized literal "likelihood").

Reply:
Static checkers can usually be configured to ignore selected warnings, I guess. I really don't think having a static checker not to complain is worth the code duplication and harder refactoring. I understand there are hundreds of this and I am not asking that they be reverted just now, but I'd honestly like to go back to the single-place definition, maybe with the shorter import as suggested above.

Reply:
(I don't mean changing them before the merge, of course, but at some point later. I am still slowly understanding the new code.)